modules/core/arrow-test/src/main/kotlin/arrow/test/laws/TraverseFilterLaws.kt

```diff
@@ -3,23 +3,30 @@ package arrow.test.laws
 import arrow.Kind
 import arrow.core.None
 import arrow.core.Some
-import arrow.typeclasses.TraverseFilter
 import arrow.test.generators.applicative
 import arrow.test.generators.functionAToB
 import arrow.test.generators.intSmall
 import arrow.typeclasses.Applicative
 import arrow.typeclasses.Eq
+import arrow.typeclasses.TraverseFilter
 import io.kotlintest.properties.Gen
 import io.kotlintest.properties.forAll
 
 object TraverseFilterLaws {
 
   // FIXME(paco): TraverseLaws cannot receive AP::just due to a crash caused by the inliner. Check in TraverseLaws why.
-  fun <F> laws(TF: TraverseFilter<F>, GA: Applicative<F>, cf: (Int) -> Kind<F, Int>, EQ: Eq<Kind<F, Int>>, EQ_NESTED: Eq<Kind<F, Kind<F, Int>>> = Eq.any()): List<Law> =
-    TraverseLaws.laws(TF, GA, cf, EQ) + listOf(
-      Law("TraverseFilter Laws: Identity") { TF.identityTraverseFilter(GA, EQ_NESTED) },
-      Law("TraverseFilter Laws: filterA consistent with TraverseFilter") { TF.filterAconsistentWithTraverseFilter(GA, EQ_NESTED) }
-    )
+  fun <F> laws(
+    TF: TraverseFilter<F>,
+    GA: Applicative<F>,
```

Comment:
Not sure about hierarchies but isn't `TraverseFilter<F>` also an `Applicative<F>`?

Reply:
i checked the source. no, it does not seem to be the case